examples/primitives-boxes/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Cubes</title>
+    <meta name="description" content="Cubes - A-Frame">
+    <script src="../../dist/aframe.js"></script>
+  </head>
+  <body>
+    <a-scene>
+
+      <a-entity position="0 0 -7" rotation="0 45 0">
+
+        <a-box src="http://i.imgur.com/6qHKYGf.png" position="-5 0 -2" rotation="0 0 0" width="1" color="#F16745"></a-box>
+        <a-box position="0 0 0" rotation="0 0 0" height="2" color="#FFC65D"></a-box>
+        <a-box position="5 0 2" rotation="0 0 0" depth="5" color="#7BC8A4"></a-box>
+        <a-box position="5 5 -5" rotation="0 0 0" width="1" height="2" depth="22" color="#4CC3D9"></a-box>
+        <a-box position="1 2 1" rotation="0 0 0" width="1" height="1" depth="1" color="#93648D"></a-box>
+        <a-box position="-3 0 0" rotation="0 0 0" width="0.5" height="10" depth="0.5" color="#404040"></a-box>
+
+        <a-frame position="-2 5 4" scale="0.15 0.15 0.15" rotation="0 0 0" thickness="0.1" color="#999"></a-frame>
```

Comment:
feel free to remove this and [that one](#diff-fd6c52500482671540e1f39d3870e749R34) if we're removing the `<a-frame>` template